### PR TITLE
Update eve api usb+can

### PR DESCRIPTION
**Pull request by christoph-zededa**
to include https://github.com/lf-edge/eve-api/pull/45

### CODE REVIEW

**Comment by milan-zededa** [comment]
It seems that you forgot to update the vendor folders with these changes.

File: Makefile [hunk 1]
```diff
@@ -834,6 +834,11 @@ cache-export-docker-load-all: $(LINUXKIT) $(addsuffix -cache-export-docker-load,
 proto-vendor:
 	@$(DOCKER_GO) "cd pkg/pillar ; go mod vendor" $(CURDIR) proto
 
+bump-eve-api:
+	find . -type f -name "go.mod" -exec grep -q 'github.com/lf-edge/eve-api/go' {} \; -execdir go get -u github.com/lf-edge/eve-api/go \;
+	find . -type f -name "go.mod" -exec grep -q 'github.com/lf-edge/eve-api/go' {} \; -execdir go mod tidy \;
```
Comment (rene) [inline Makefile:839]: You don't need to execute 3x the find command and look 3x for all files, you can just pass multiple -execdir arguments, like this:

```
find . -type f -name "go.mod" -exec grep -q 'github.com/lf-edge/eve-api/go' {} \; -execdir go get -u github.com/lf-edge/eve-api/go \; -execdir go mod tidy \; -execdir go mod vendor \;
```

File: Makefile [hunk 2]
```diff
@@ -834,6 +834,9 @@ cache-export-docker-load-all: $(LINUXKIT) $(addsuffix -cache-export-docker-load,
 proto-vendor:
 	@$(DOCKER_GO) "cd pkg/pillar ; go mod vendor" $(CURDIR) proto
 
+bump-eve-api:
```
Comment (rene) [inline Makefile:837]: @christoph-zededa , I forgot to mention that would be good to have this target explained in the `make help` target....

Comment (christoph-zededa) [reply Makefile:837]: done; please have a look at the help text

**Review by rene — APPROVED** [review]
Thanks @christoph-zededa , LGTM.

**Review by eriknordmark — APPROVED** [review]
Looks good

**Pull request merged**